documentation/configure.markdown

```diff
@@ -177,6 +177,8 @@ When this is set to true, debugging information will be written to `<vardir>/deb
 
 ### `url-prefix`
 
+> **Deprecated:** This setting should not be necessary in most cases and this kind of configuration is done different by most other Puppet Labs services.
```

Comment:
`done differently`? Anyway ... do we need to say all this? Maybe just say its going and leave it at that.